app/utils.js

```diff
@@ -9,3 +11,10 @@ export const isBrowser = () => {
 
 // Return formatted full name of user
 export const fullName = user => `${user.first_name} ${user.last_name}`;
+
+// Redirect page for logged in user based on role
+export const loginRedirect = role => {
+  if (role === THERAPIST) return '/dashboard';
+  else if (role === PATIENT) return '/plan';
+  else return '/';
```

Comment:
does the therapist have a dashboard?  I thought their redirect page on the mvp was the patient list (with the add patient button)